docs/make.jl

```diff
@@ -6,17 +6,18 @@ makedocs(;
     authors="Ole Kröger <o.kroeger@opensourc.es> and contributors",
     repo="https://github.com/Wikunia/Javis.jl/blob/{commit}{path}#L{line}",
     sitename="Javis.jl",
-    format=Documenter.HTML(;
-        prettyurls=get(ENV, "CI", "false") == "true",
-        canonical="https://Wikunia.github.io/Javis.jl",
-        assets=String[],
-    ),
+    #format=Documenter.HTML(;
+    #    prettyurls=get(ENV, "CI", "false") == "true",
+    #    canonical="https://Wikunia.github.io/Javis.jl",
+    #    assets=String[],
+    #),
     pages=[
         "Home" => "index.md",
         "References" => "references.md",
+        "Contributing" => "contributing.md",
     ],
 )
 
-deploydocs(;
-    repo="github.com/Wikunia/Javis.jl",
-)
+#deploydocs(;
+#    repo="github.com/Wikunia/Javis.jl",
+#)
```

Comment:
This is needed to deploy the docs via GitHub actions and should not be "deleted". It probably throws a warning locally but that's normal.

Reply:
I see. I thought the CI was causing the issue as when I looked into it online, they mentioned CI caused the error.

``
┌ Warning: Documenter could not auto-detect the building environment Skipping deployment.
└ @ Documenter ~/.julia/packages/Documenter/bRylW/src/deployconfig.jl:75
``